src/System.Runtime.Caching/src/System/Runtime/Caching/MemoryCache.cs

```diff
@@ -28,7 +28,7 @@ public class MemoryCache : ObjectCache, IEnumerable, IDisposable
                                                               | DefaultCacheCapabilities.CacheEntryRemovedCallback;
         private static readonly TimeSpan s_oneYear = new TimeSpan(365, 0, 0, 0);
         private static object s_initLock = new object();
-        private static MemoryCache s_defaultCache;
+        private static volatile MemoryCache s_defaultCache;
```

Comment:
Rather than making this volatile, I'd prefer to see the Default property changed to just be:
```C#
public static MemoryCache Default => LazyInitializer.EnsureInitialized(ref s_defaultCache);
```